modules/message-broker-api/src/main/java/org/opencastproject/message/broker/api/series/SeriesItem.java

```diff
@@ -51,6 +51,7 @@
   private final Boolean optOut;
   private final String element;
   private final String elementType;
+  private final Boolean overrideEpisodeAcl;
```

Comment:
This, unfortunately, broke sending any series related messages in develop.
The issue is fixed by pull request #659.
Please avoid using any Objects except Strings in these message items.

Reply:
Thanks for noticing, I'll try to keep this in mind for the future.